server/routes/billingRoutes.js

```diff
@@ -1,5 +1,5 @@
 const express = require('express');
-const stripe = require('stripe')("sk_test_WjgLJcjcHXfmcGedEp7NrXRi"); // test key
+const stripe = require('stripe')(process.env.REACT_APP_SECRET_KEY); // secret test key in .env
```

Comment:
I think since this is on the backend, you don't need the REACT_APP at the beginning of this. Did you set an .env variable in the server directory? If not, that might be the cause of one of the errors.